stl/src/xstrxfrm.cpp

```diff
@@ -53,37 +53,38 @@ _EXTERN_C_UNLESS_PURE
 //     Non-standard: if OM/API error, return INT_MAX.
 _CRTIMP2_PURE size_t __CLRCALL_PURE_OR_CDECL _Strxfrm(
     char* _string1, char* _end1, const char* _string2, const char* _end2, const _Collvec* ploc) {
-    size_t _n1 = _end1 - _string1;
-    size_t _n2 = _end2 - _string2;
-    int dstlen;
-    size_t retval = (size_t) -1; // NON-ANSI: default if OM or API error
+    size_t _n1    = _end1 - _string1;
+    size_t _n2    = _end2 - _string2;
+    size_t retval = static_cast<size_t>(-1); // NON-ANSI: default if OM or API error
     UINT codepage;
     const wchar_t* locale_name;
 
-    if (ploc == 0) {
+    if (ploc == nullptr) {
         locale_name = ___lc_locale_name_func()[LC_COLLATE];
         codepage    = ___lc_collate_cp_func();
     } else {
         locale_name = ploc->_LocaleName;
         codepage    = ploc->_Page;
     }
 
-    if ((locale_name == nullptr) && (codepage == CP_ACP)) {
+    if (locale_name == nullptr && codepage == CP_ACP) {
         if (_n2 <= _n1) {
             memcpy(_string1, _string2, _n2);
         }
         retval = _n2;

```

Comment:
Early return?

Yes, I'm creeping scope even more; feel free to disregard. The contorted "single-return" control-flow here gets to me. The danger of touching any of the crufty files in `src` is that we always want to rewrite them completely.

Reply:
While I agree in principle, I'm going to resolve control flow surgery as wontfix for this PR. I find control flow surgery the hardest to review in others' PRs (i.e. the most likely to introduce damage) and we shouldn't be spending too much time away from C++20 features right now.